service/controller/app/v1/resource/chart/desired_test.go

```diff
@@ -221,30 +222,31 @@ func Test_processLabels(t *testing.T) {
 	tests := []struct {
 		name           string
 		projectName    string
+		chartVersion   string
 		inputLabels    map[string]string
 		expectedLabels map[string]string
 	}{
 		{
-			name:        "case 0: basic match",
-			projectName: "app-operator",
+			name:         "case 0: basic match",
+			projectName:  "app-operator",
+			chartVersion: "1.0.0",
 			inputLabels: map[string]string{
-				"app-operator.giantswarm.io/version": "1.0.0",
-				"giantswarm.io/managed-by":           "release-operator",
+				"giantswarm.io/managed-by": "release-operator",
 			},
 			expectedLabels: map[string]string{
 				"chart-operator.giantswarm.io/version": "1.0.0",
 				"giantswarm.io/managed-by":             "app-operator",
 			},
 		},
 		{
-			name:        "case 1: extra labels still present",
-			projectName: "app-operator",
+			name:         "case 1: extra labels still present",
+			projectName:  "app-operator",
+			chartVersion: "1.0.0",
 			inputLabels: map[string]string{
-				"app":                                "prometheus",
-				"app-operator.giantswarm.io/version": "1.0.0",
-				"giantswarm.io/cluster":              "5xchu",
-				"giantswarm.io/managed-by":           "cluster-operator",
-				"giantswarm.io/organization":         "giantswarm",
+				"app":                        "prometheus",
+				"giantswarm.io/cluster":      "5xchu",
+				"giantswarm.io/managed-by":   "cluster-operator",
```

Comment:
```suggestion
				"giantswarm.io/managed-by":   "app-operator",
```

Reply:
I'm moving it to new file `lable_test.go`